ttyplot.c

```diff
@@ -199,7 +199,8 @@ void paint_plot() {
     sigprocmask(SIG_UNBLOCK, &sigmsk, NULL);
 }
 
-void resize() {
+void resize(int signum) {
+    (void)signum;
```

Comment:
OK I understand that a handler can receive int signal number, but I'm not sure what does signum on line 203 do?

Reply:
Now answered at https://github.com/tenox7/ttyplot/pull/70#issuecomment-1465485595 (for anyone else coming here)